docs/view-style-props.md

```diff
@@ -309,6 +309,35 @@ Sets the elevation of a view, using Android's underlying [elevation API](https:/
 
 ---
 
+### `filter`
+
+Adds a graphical filter to the View. This filter is comprised of any number of _filter functions_, which each represent some atomic change to the graphical composition of the View. The complete list of valid filter functions is defined below. `filter` will apply to descendants of the View as well as the View itself. `filter` implies `overflow: hidden`, so descendants will be clipped to fit the bounds of the View.
+
+The following filter functions work across all platforms:
+
+- `brightness`: Changes the brightness of the View. Takes a non-negative number or percentage.
+- `opacity`: Changes the opacity, or alpha, of the View. Takes a non-negative number or percentage.
+
+<div class="label basic android">Android</div>
+
+The following filter functions work on Android only:
+
+- `blur`: Blurs the View with a [Guassian blur](https://en.wikipedia.org/wiki/Gaussian_blur), where the specified length represents the radius used in the blurring algorithm. Any non-negative DIP value is valid (no percents). The larger the value, the blurrier the result.
```

Comment:
```suggestion
- `blur`: Blurs the `View` with a [Guassian blur](https://en.wikipedia.org/wiki/Gaussian_blur), where the specified length represents the radius used in the blurring algorithm. Any non-negative DIP value is valid (no percents). The larger the value, the blurrier the result.
```

Reply:
All mention of `View` should be backticked in the page as it is a proper RN component.